### ISSUE TITLE
LavinMQ crashing during startup because of "Lease lost <number>"

### ISSUE THREAD

**Issue by viktorerlingsson** [cloudamqp/lavinmq#829]
**Describe the bug**
LavinMQ failed to start on a single node cluster with a "Lost lease" error from `lease_keepalive`. 

```
Nov 07 10:35:48 systemd Starting LavinMQ...
...
<launching, loading data...>
...
Nov 07 10:35:55 lavinmq lmq.etcd Lost leadership of lavinmq/leader: Lost lease 3008003254974389534
....
Nov 07 10:35:58 lavinmq lmq.launcher Lost leadership lease
Nov 07 10:35:58 lavinmq lmq.launcher Stopping
```

This unexpected shutdown during the startup then caused other errors in the startup, and left the LavinMQ service in an inactive (dead) state. 

Logs from after "Stopping": 
```
Nov 07 10:35:58 lavinmq Unhandled exception in spawn(name: HTTP listener): Can't re-start closed server (Exception)
Nov 07 10:35:58 lavinmq   from usr/share/crystal/src/http/server.cr:457:5 in 'listen'
Nov 07 10:35:58 lavinmq   from usr/share/crystal/src/fiber.cr:143:11 in 'run'
Nov 07 10:35:58 lavinmq   from ???
Nov 07 10:35:58 systemd Started LavinMQ.
```

Service status
```
lavinmq.service - LavinMQ
     Loaded: loaded (/lib/systemd/system/lavinmq.service; enabled; vendor preset: enabled)
     Active: inactive (dead) since Thu 2024-11-07 09:03:28 UTC; 4min 58s ago
       Docs: https://github.com/cloudamqp/lavinmq
    Process: 988772 ExecStart=/usr/bin/lavinmq -c /etc/lavinmq/lavinmq.ini (code=exited, status=0/SUCCESS)
   Main PID: 988772 (code=exited, status=0/SUCCESS)
```

**Describe your setup**
One single node running ubuntu 22.04.
LavinMQ v2.0.0 and etcd 3.5.16 running on the same machine. 

**How to reproduce**
Have not been able to reproduce it again. 

**Expected behavior**
LavinMQ starts, gets lease from etcd and keeps running.

**Comment by viktorerlingsson** [comment]
The problem is probably that the lease TTL expires during startup if the startup takes more than 5s, leading to there being no lease when LavinMQ has actually started.

**Comment by viktorerlingsson** [comment]
Fixed by #834
